cloudformation/panther-deployment-role.yml

```diff
@@ -212,6 +212,7 @@ Resources:
               - dynamodb:CreateTable
               - dynamodb:DeleteTable
               - dynamodb:Describe*
+              - dynamodb:Scan
```

Comment:
These come from this PR https://github.com/panther-labs/panther-auxiliary/pull/64/files